docs/metrics.md

```diff
@@ -24,6 +24,48 @@ of the last second of activity, whereas the 1 minute intervals are used for
 reporting values to an external database in which case the values reported are
 all reflective of the last minute of activity.
 
+### Default Metrics
+
+Colossus makes several metrics available in a default service, they are:
+
+* *service name*/system/fd_count
+* *service name*/system/gc/cycles
+* *service name*/system/gc/msec
+* *service name*/system/memory
+* *service name*/requests
+* *service name*/concurrent_requests
+* *service name*/requests_per_connection
+* *service name*/errors
+* *service name*/dropped_requests
+* *service name*/connection_failures
+* *service name*/disconnects
+* *service name*/latency
+* *service name*/transit_time
+* *service name*/queue_time
+* *service name*/worker/event_loops
+* *service name*/worker/connections
+* *service name*/worker/rejected_connections
+* *service name*/connections
+* *service name*/refused_connections
+* *service name*/connects
+* *service name*/closed
+* *service name*/highwaters
+
+
+### Metric Types
```

Comment:
I think this section is kinda redundant with the "Available Collectors" section below.